src/expressions/operator-expr.md

```diff
@@ -60,8 +82,25 @@ let mut array = [-2, 3, 9];
 }
 ```
 
+For ease of typing, consecutive borrow expressions can be expressed by the
```

Comment:
Ease of typing compared to what? To most people reading the reference, it would seem obvious that `&&` is allowed and can be chained just like e.g. `--1` works (though is useless). Though I'm guessing that the `x && y` operator giving an `&&` token meaning is to blame here?